packages/cfd/src/features/Components/copy-text-component.tsx

```diff
@@ -10,7 +10,7 @@ type TCopyTextComponent = {
 
 const CopyTextComponent = ({ text, className }: TCopyTextComponent) => {
     return (
-        <div className={className} data-testid='cfd_account_copy_main_div'>
+        <div className={className} data-testid='copy_text_component_main_div'>
```

Comment:
should we follow one convention for test ids like we are following 
`dt_test_id`
```suggestion
        <div className={className} data-testid='dt_copy_text_component_main_div'>
```